frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -194,13 +194,18 @@ export const CommandBar = () => {
         },
     );
     useKeyboardShortcut({ key: 'Escape' }, () => {
-        if (searchContainerRef.current?.contains(document.activeElement)) {
-            searchInputRef.current?.blur();
-        }
+        setShowSuggestions(false);
     });
     const placeholder = `Command bar (${hotkey})`;
 
     useOnClickOutside([searchContainerRef], hideSuggestions);
+    const onKeyDown = (event: any) => {
+        if (event.key === 'Escape') {
+            setShowSuggestions(false);
+        } else if (event.keyCode >= 48 && event.keyCode <= 110) {
```

Comment:
Anything outside these we should be interested in accepting?

Reply:
Maybe this event: React.KeyboardEvent<HTMLSpanElement>

Reply:
Cool thanks, I wasn't aware there was a difference between React.KeyboardEvent and KeyboardEvent. That works! I'll push that improvement and merge